BetterThanRottenTomatoAPI/ReviewManagement/ReviewManagement.Domain/Services/ReviewService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using ReviewManagement.Domain.Interfaces;
+using ReviewManagement.Domain.Models;
+using ReviewManagement.Infrastructure.Repositories.Interfaces;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ReviewManagement.Domain.Mappers;
+using System;
+
+namespace ReviewManagement.Domain.Services
+{
+    public class ReviewService : IReviewService
+    {
+        private readonly IReviewRepository _reviewRepository;
+
+        public ReviewService(IReviewRepository reviewRepo)
+        {
+            _reviewRepository = reviewRepo;
+        }
+        public async Task AddMovieReviewService(MovieReviewModel movieReview)
+        {
+            if(movieReview == null)
+            {
+                throw new ArgumentException("Movie Review not provided");
+            }
+            var dbMovieReview = ReviewMappers.ReviewModelToReviewEntity(movieReview);
+            await _reviewRepository.AddMovieReview(dbMovieReview);
+        }
+
+        public async Task<List<MovieReviewModel>> GetMovieReviewsByMovieIdService(long movieId)
+        {
+            if(movieId <= 0)
+            {
+                throw new ArgumentException("Movie Id not found");
+            }
+            var movieReviewModelList = new List<MovieReviewModel>();
+
+            var dbReviewList = await _reviewRepository.GetMovieReviewsByMovieId(movieId);
+
+            if(dbReviewList == null)
+            {
+                throw new Exception("No reviews found");
```

Comment:
I think if no reviews found, its not an exception, just return an empty list. If you throw exception, that can result in error in u.i, but it is not error, just no reviews have been made yet.